css/index.css

```diff
@@ -1,66 +1,213 @@
 
+
+
+  /* Base styles */
 body {
-    background-color: rgb(142, 142, 158);
-    color: white;
-    justify-content: center;
-    align-items: center;
-    height: 100vh;
     margin: 0;
-    padding-left: 50px;
-    padding-top: 100px;
-    padding-bottom: 50px;
-    padding-right: 60px;
-    /* text-align: center; */
-    /* display: flex; */
-  }
-  .wrapper { 
-    margin: auto;
-    width: 1000px;
+    font-family: Roboto, sans-serif;
+    font-size: 16px; /* Adjust base font size */
+    line-height: 1.5; /* Improve readability */
   }
-.img-coding { 
-width: 400px;
-height: 200px;
+
+  /* Desktop first layout*/
+html { 
+    scroll-behavior: smooth;
+}
+header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    z-index: 999;
 }
 
-h1 { 
+.header {
+    padding: 40px;
+    text-align: center;
+    background: gold;
+    margin: 30px;
     font-size: 20px;
+  }
+
+#skills ul {
+    padding-left: 10px;
+    background-color: lightcoral;
+  }
+
+   #About h2 {
+    font-size: 60px; 
+    font-style: italic; 
     color: green;
+  }
+  .image-container {
+    width: 650px; 
+    height: 250px; 
+    border-radius: 50%;
+    overflow: hidden;
+    margin: 0 auto; 
+}
+
+.profile-image {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
 }
 
+
+body { 
```

Comment:
This `body` section and the one at the top of the document are both being applied to the default since neither is being wrapped in a media tag.